cmd/fyne/internal/commands/build.go

```diff
@@ -282,6 +287,25 @@ func (b *Builder) build() error {
 	return err
 }
 
+func (b *Builder) computeSrcDir(fyneGoModRunner runner) (string, error) {
+	if b.goPackage == "" {
+		return b.srcdir, nil
+	}
+
+	srcdir := b.srcdir
+	if strings.HasPrefix(b.goPackage, "./") {
```

Comment:
This doesn't seem right? `./` is relative to the current directory rather than a declared source directory isn't it?

Reply:
Also what about the "." case? That is simply cwd.

Reply:
go build take a module path. It can be relative, in which case it start in the current location or it will look up in GOROOT for it.

Reply:
> If we want to match how go compiler handles it can we use their APIs instead of trying to work it out?

I don't see any API for the purpose of finding where go build is going to look where the source it is asked to build are. Did you have one in mind?